contracts/ens/ENSRegistry.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity ^0.4.24;
+
+import "./ENS.sol";
+
+/**
+  * @title The ENS registry contract
+*/
+contract ENSRegistry is ENS {
```

Comment:
Are the ENS contracts not available in an existing repository somewhere? Did you hand code this or copy it from somewhere? Seems weird that we would have to have our own implementation of ENS in here.

Reply:
No hand code, stripped bare minimum of what I needed from https://github.com/ensdomains/ens/tree/master/contracts